### ISSUE TITLE
Vietnam exhibit page- deprecated.openvault links

### ISSUE THREAD

**Issue by Muraszko** [afred/openvault#1139]
Trying to find a useful way to populate the Vietnam exhibit page, since much of the media that belongs there does not have the proper pbcoreRelation data.  You'll see on deprecated.openvault, under the Vietnam Collection there were assets that were in that collection that had no relationship data, but were assigned to be found within that collection.

On deprecated.openvault records were assigned to a specific collection (War and Peace, Vietnam, etc.) via command line during the ingest process.  So in Solr there is collection information that was assigned by the person ingesting.  So that "relationship" exists only there, but we can still find it using the old collection page's canned searches.

Basically, I'm thinking we use deprecated.openvault canned searches to find the group of assets that belong in the Vietnam collection.  Grab their UOI_IDs, which should match current Open Vault asset UOI_IDs and use those sets to populate the new exhibit page. 

Below are the Vietnam Collection canned searches that were on the collection page on deprecated.openvault:

```
<li><a href="http://openvault.wgbh.org/catalog?f[ri_collection_ancestors_s][]=info:fedora/org.wgbh.mla:vietnam">All materials</a></li>
<li><a href="http://openvault.wgbh.org/catalog?f[ri_collection_ancestors_s][]=info:fedora/org.wgbh.mla:vietnam&amp;f%5Btopic_cv%5D%5B%5D=Vietnam+War%2C+1961-1975--Personal+narratives">Interview video, audio, and transcripts</a></li>
<li><a href="http://openvault.wgbh.org/catalog?f[ri_collection_ancestors_s][]=info:fedora/org.wgbh.mla:vietnam&amp;f%5Bdc_type_s%5D%5B%5D=Stock+footage">News video and archival footage </a></li>
<li><a href="http://openvault.wgbh.org/catalog?f[ri_collection_ancestors_s][]=info:fedora/org.wgbh.mla:vietnam&amp;f%5Bmedia_s%5D%5B%5D=Image">Photographs</a></li>
<li><a href="http://openvault.wgbh.org/catalog?f%5Bri_collection_ancestors_s%5D%5B%5D=info%3Afedora%2Forg.wgbh.mla%3Avietnam&amp;f%5Buois_item_category_s%5D%5B%5D=Vietnam%20original%20b-roll%20footage">Original video footage</a></li>
```

The searches still work, you just had to replace the beginning of the URL with deprecated.openvault.wgbh.org:
http://deprecated.openvault.wgbh.org/catalog?f[ri_collection_ancestors_s][]=info:fedora/org.wgbh.mla:vietnam

So using those searches, can you guys harvest the UOI_IDs and use those to populate the new Open Vault Vietnam exhibit page?  Is that the best way to do it? 
If so, be sure to talk to Kevin, he has many scripts that are useful in harvesting the old site for UOI_IDs from searches.

**Comment by Muraszko** [comment]
Try and keep current exhibit page tabs intact.

**Comment by Muraszko** [comment]
I have an email out to Kevin to see if he can grab the UOI_IDs from the old site, stand by.

**Comment by Muraszko** [comment]
@afred 
I have four files with all the Vietnam exhibit page UOI_IDs, broken down by tab, find them here:
smb://deptfs.wgbh.org/dept/MLA/OpenVault_Upload/Vietnam_Exhibit_UOI_IDs

**Comment by afred** [comment]
closed by #1208
